website/source/docs/agent/configuration/server.html.md

```diff
@@ -0,0 +1,183 @@
+  cluster again when starting. This flag allows the previous state to be used to
+  rejoin the cluster.
+
+- `retry_join` `(array<string>: [])` - Specifies a list of server addresses to
+  retry joining if the first attempt fails. This is similar to
+  [`start_join`](#start_join), but only invokes if the initial join attempt
+  fails. The list of addresses will be tried in the order specified, until one
+  succeeds. After one succeeds, no further addresses will be contacted. This is
+  useful for cases where we know the address will become available eventually.
+  Use `retry_join` with an array as a replacement for `start_join`, **do not use
+  both options**. See the [server address format](#server-address-format)
+  section for more information on the format of the string.
+
+- `retry_interval` `(string: "30s")` - Specifies the time to wait between retry
+  join attempts.
+
+- `retry_max` `(int: 0)` - Specifies the maximum number of join attempts to be
+  made before exiting with a return code of 1. By default, this is set to 0
+  which is interpreted as infinite retries.
+
+- `start_join` `(array<string>: [])` - Specifies a list of server addresses to
+  join on startup. If Nomad is unable to join with any of the specified
+  addresses, agent startup will fail. See the
+  [server address format](#server-address-format) section for more information
+  on the format of the string.
+
+### Server Address Format
+
+This section describes the acceptable syntax and format for describing the
+location of a Nomad server. There are many ways to reference a Nomad server,
+including directly by IP address and resolving through DNS.
+
+#### Directly via IP Address
+
+It is possible to address another Nomad server using its IP address. This is
+done in the `ip:port` format, such as:
+
+```
+1.2.3.4:5678
+```
+
+If the port option is omitted, it defaults to the Serf port, which is 4648
+unless configured otherwise:
+
+```
+1.2.3.4 => 1.2.3.4:4648
+```
+
+#### Via Domains or DNS
+
+It is possible to address another Nomad server using its DNS address. This is
+done in the `address:port` format, such as:
+
+```
+nomad-01.company.local:5678
+```
+
+If the port option is omitted, it defaults to the Serf port, which is 4648
+unless configured otherwise:
+
+```
+nomad-01.company.local => nomad-01.company.local:4648
+```
+
+## `server` Examples
+
+### Common Setup
+
+This example shows a common Nomad agent `server` configuration stanza. The two
+IP addresses could also be DNS, and should point to the other Nomad servers in
+the cluster
+
+```hcl
+server {
+  enabled          = true
+  bootstrap_expect = 3
+  retry_join       = ["1.2.3.4", "5.6.7.8"]
+}
+```
+
+### Automatic Bootstrapping
+
+The Nomad servers can automatically bootstrap if Consul is configured. For a
+more detailed explanation, please see the
+[automatic Nomad bootstrapping documentation](/docs/cluster/automatic.html).
+
+### Restricting Schedulers
```

Comment:
Would prefer one setting data dir. This example will be less useful as no one will really restrict